py/desispec/test/test_brick.py

```diff
@@ -1,31 +1,53 @@
-import unittest
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""
+desispec.test.test_brick
+========================
 
-import desispec.brick
+Test desispec.brick.
+"""
+from __future__ import absolute_import, unicode_literals
+# The line above will help with 2to3 support.
+import unittest
 import numpy as np
+from ..brick import brickname
+
 
 class TestBrick(unittest.TestCase):
-    
+    """Test desispec.brick.
+    """
+
     def setUp(self):
         n = 10
         self.ra = np.linspace(0, 3, n) - 1.5
         self.dec = np.linspace(0, 3, n) - 1.5
         self.names = np.array(
             ['3587m015', '3592m010', '3597m010', '3597m005', '0002p000',
             '0002p000', '0007p005', '0007p010', '0012p010', '0017p015'])
-            
-    def test1(self):
-        brickname = desispec.brick.brickname(0, 0)
-        self.assertEqual(brickname, '0002p000')
 
-    def test2(self):
-        brickname = desispec.brick.brickname(self.ra, self.dec)
-        self.assertEqual(len(brickname), len(self.ra))
-        self.assertTrue(np.all(brickname == self.names))
+    def test_brickname_scalar(self):
+        """Test a scalar brick name conversion.
+        """
+        b = brickname(0, 0)
+        self.assertEqual(b, '0002p000')
+
+    def test_brickname_array(self):
+        """Test an array brick name conversion.
+        """
+        bricknames = brickname(self.ra, self.dec)
+        self.assertEqual(len(bricknames), len(self.ra))
+        self.assertTrue((bricknames == self.names).all())
 
     def test3(self):
-        brickname = desispec.brick.brickname(np.array(self.ra), np.array(self.dec))
-        self.assertEqual(len(brickname), len(self.ra))
-        self.assertTrue(np.all(brickname == self.names))
-                
-if __name__ == '__main__':
-    unittest.main()
+        """This test is nonsensical, because it is a repeat of test2.
```

Comment:
This is just a repeat of `test_brickname_array`.  Specifically, since `self.ra` and `self.dec` are *already* arrays, calling `np.array()` on them does nothing.